### ISSUE TITLE
[Find/Replace] [Dark Theme] Bad contrast on some buttons

### ISSUE THREAD

**Issue by nlisker** [eclipse-platform/eclipse.platform.ui#2284]
![image](https://github.com/user-attachments/assets/42ef93d8-52a9-41b9-b9b0-1b6fb466b28d)

Except for the 3 colored icons, the gray on dark gray icons don't provide good contrast. Making them white-ish like the text, or colored like the other 3 are two satisfactory solutions I can think of.

**Comment by nlisker** [comment]
Should be added to #2021?

**Comment by HeikoKlare** [comment]
I agree that these icons are suboptimal. The problem is that icons cannot be themed, so we need to choose icons that fit for both light and dark theme(s). The chosen greyish color is the best trade-off we were able to find that made them usable in both light and dark theme. Using icon fonts might be a good solution, but we already tried that for the expand button (the ">" to open the replace bar) and faced some issues, which is why we moved back to ordinary icons/images.
Any proposals on how to improve this are highly appreciated.

**Comment by nlisker** [comment]
Here are 3 possible solutions:
1. Use icons like you do for the other 3. If it works there it will work here. What issues did you hit? Put the expand button aside for now, there are 6 buttons which can be solved this way.
2. Use a contrasting color outline: dark fill with light outline or vice versa. The icons are small and delicate so an outline might make them more bulky, but it's a known technique.
3. Use a non-grayscale color. Blue tends to be neutral in UI's. Find a blue hue that works well in both cases, probably a light blue or even desaturated cyan, just as examples.
